message MSG_CLIENT_DM_DOMAIN_CREATE
{
    optional string token = 1;
    optional string domainName = 2;
    optional string domainDescription = 3;
    optional double coreNum = 4;
    optional uint32 memSize = 5;
}

message MSG_DM_CLIENT_DOMAIN_CREATE_ACK
{
    optional int32 statusCode = 1;
}